csm/cli/schema/csm_setup.json

```diff
@@ -246,40 +246,10 @@
       "permissions_tag": "update",
       "args": [
         {
```

Comment:
There is an entry for refresh_config which is not required. Remove it.

Reply:
Modifications done